backend/src/test/java/com/woowacourse/levellog/presentation/ControllerTest.java

```diff
@@ -109,38 +118,43 @@ void setUp(final WebApplicationContext context, final RestDocumentationContextPr
                 ).build();
     }
 
-    protected ResultActions requestGet(final String url, final String token) throws Exception {
+    private void mockLogin() {
+        given(jwtTokenProvider.getPayload(VALID_TOKEN)).willReturn("1");
+        given(jwtTokenProvider.validateToken(VALID_TOKEN)).willReturn(true);
+    }
+
+    protected ResultActions requestGet(final String url) throws Exception {
         return mockMvc.perform(get(url)
-                        .header(HttpHeaders.AUTHORIZATION, "Bearer " + token)
+                        .header(HttpHeaders.AUTHORIZATION, "Bearer " + VALID_TOKEN)
                         .contentType(MediaType.APPLICATION_JSON))
                 .andDo(print());
     }
 
-    protected ResultActions requestPost(final String url, final String token, final Object request)
+    protected ResultActions requestPost(final String url, final Object request)
             throws Exception {
```

Comment:
```suggestion
    protected ResultActions requestPost(final String url, final Object request) throws Exception {
```
GET, DELETE는 개행을 안 했는데 POST, PUT에서만 하고 있네용